src/components/recruting/_02_prepare/_02/GroupIdeal.tsx

```
@@ -1,51 +1,53 @@
 import { useForm } from "react-hook-form";
 import { useGroupStore } from "../../../../store/useStore";
 
 export default function GroupIdeal() {
   const { group: groups } = useGroupStore();
 
-  const [groupIdeals, setGroupIdeals] = useState<GroupIdeals>(() =>
-    groups.reduce(
+  const [groupIdeals, setGroupIdeals] = useState<GroupIdeals>(() => {
+    return groups.reduce(
       (acc, group) => ({
         ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)